examples/get_transactions/main.go

```diff
@@ -22,20 +22,22 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/onflow/flow-go-sdk/access/http"
+
 	"github.com/onflow/flow-go-sdk"
-	"github.com/onflow/flow-go-sdk/client"
 	"github.com/onflow/flow-go-sdk/examples"
-	"google.golang.org/grpc"
 )
 
 func main() {
-	txID := prepareDemo()
-	demo(txID)
+	demo()
 }
 
-func demo(txID flow.Identifier) {
+func demo() {
 	ctx := context.Background()
-	flowClient := examples.NewFlowClient()
+	flowClient, err := http.NewClient(http.EmulatorHost)
```

Comment:
How does the client get closed? Does it not need to be?

Reply:
Good flag. Close method is planned to be added to the interface.